src/Passes/Source/Commandline/ConfigBind.hpp

```diff
@@ -0,0 +1,207 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Commandline/ConfigurationBindInterface.hpp"
+#include "Commandline/ParameterParser.hpp"
+
+#include "Llvm/Llvm.hpp"
+
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <type_traits>
+#include <typeindex>
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    /// Generic implementation of the bind interface for different types. This class holds the reference
+    /// to the variable and the name of the parameter. It implements serialisers and deserialisers to
+    /// allow transforming strings to native values and vice versa.
```

Comment:
> This class holds the reference to the **variable** and the name of the **parameter**.

Do you mean the "command line parameter" and the "variable corresponding to it"?

Reply:
I've changed the wording accordingly. Note that it holds a reference to the variable and not the variable itself. The variable itself is held by the configuration class.